imgui/imgui.h

```diff
@@ -936,7 +946,7 @@ enum ImGuiInputTextFlags_
     ImGuiInputTextFlags_AllowTabInput       = 1 << 10,  // Pressing TAB input a '\t' character into the text field
     ImGuiInputTextFlags_CtrlEnterForNewLine = 1 << 11,  // In multi-line mode, unfocus with Enter, add new line with Ctrl+Enter (default is opposite: unfocus with Ctrl+Enter, add line with Enter).
     ImGuiInputTextFlags_NoHorizontalScroll  = 1 << 12,  // Disable following the cursor horizontally
-    ImGuiInputTextFlags_AlwaysInsertMode    = 1 << 13,  // Insert mode
```

Comment:
Please add the corresponding Go constant `InputTextFlagsAlwaysOverwrite`.
In order to avoid a major versin bump, define the existing `InputTextFlagsAlwaysInsertMode` to be equal the new one, and mark it as "Deprecated", with hint of becoming removed in next major version.